src/components/CreateNewProjectWizard/SummaryPane.js

```diff
@@ -185,6 +185,29 @@ class SummaryPane extends PureComponent<Props> {
           </Fragment>
         );
       }
+      case 'projectStarter': {
+        // todo: why is a key needed on FadeIn? Was s3t.
+        // todo: should we rename projectStarter to be mores specific as this is Gatbsy only.
```

Comment:
Typo, Gatsby.

Also I don't understand the first todo, what's s3t?

Reply:
Oh, OK, I'll fix that typo.
The `key="s3t"` is in line 178 of SummaryPane. I haven't added it to the Gatsby part. Is the key needed or do I miss why it's there?